src/hpp2plantuml/hpp2plantuml.py

```diff
@@ -1033,12 +1101,12 @@ def build_aggregation_list(self):
                 comp_parent_obj = class_list_obj[comp_parent_idx]['obj']
                 self._aggregation_list.append(
                     ClassAggregationRelationship(
-                        obj_class_obj, comp_parent_obj, comp_count,
+                        comp_parent_obj, obj_class_obj, comp_count,
```

Comment:
I would suggest leaving this change out of this pull request.

I believe the current version (v0.5) is producing the correct output:

    class Engine {};
    class Car{ Engine* engine; };

produces `Car o-- Engine` as expected.

However, the current code is misleading/wrong, thanks for pointing that out.  I
am passing `obj_class_obj` as `c_parent` and `comp_parent_obj` as `c_child` when
instantiating the `ClassAggregationRelationship` object.  This is wrong and your
change is correct, but the code needs to be further modified to produce the same
output as v0.5.  I would propose removing this change and fix that in another
commit (I will take care of that before releasing v0.6).